proof-of-spacetime.md

```diff
@@ -53,7 +53,11 @@ Each reported fault carries a penality with it.
 
 ```go
 // Generate a new PoSt.
-func GeneratePoSt(sectorSize BytesAmount, sectors SectorSet, seed Seed, faults FaultSet) PoStProof {
+// - The `sectors` slice represents the sectors in a miner's proving set and
+//   must be sorted by Commitment in lexicographical order.
+// - The `faults` slice contains indices into the `sectors` slice, each
+//   representing a previously self-reported storage fault.
```

Comment:
> I agree that it's an appropriate term for discussing either Go or Rust code

I used the term "slice" because we're using Go in all of these examples. Perhaps instead I should be thinking about these pseudocode examples as using Go _syntax_, but not Go semantics.

Anyhow, I agree with your general point that this pseudocode is a little too-real to be considered pseudocode.

Reply:
> must be sorted by Commitment in lexicographical order.

ascending or descending?